pkg/kube/interface.go

```diff
@@ -68,6 +68,8 @@ type Interface interface {
 
 	// IsReachable checks whether the client is able to connect to the cluster.
 	IsReachable() error
+
+	Get(resources ResourceList, reader io.Reader) (string, error)
```

Comment:
Unfortunately, this Interface can't be changed. It would be compatibility breaking so we can't do it until Helm v4. The [HIP that describes backwards compatibility](https://github.com/helm/community/blob/main/hips/hip-0004.md) points to the [Go projects stance and method for handling this](https://go.dev/blog/module-compatibility#working-with-interfaces).

If you look in this file you'll see we're done this before with `InterfaceExt`. We'll need another new one like `InterfaceGet` or something similar. Could you please add the comment about merging this with the main interface for Helm v4, like `InterfaceExt` does.

Also, could you please document the function.